src/core/projectinfo.cpp

```diff
@@ -28,6 +28,8 @@ ProjectInfo::ProjectInfo( QObject *parent )
   : QObject( parent )
 {
   mSaveExtentTimer.setSingleShot( true );
+  mSaveRotationTimer.setSingleShot( true );
```

Comment:
@m-kuhn , found this issue while dissecting why Qt6 builds were showing some brief pauses every ~1sec when dragging the canvas. Turns out we were running these timers at infinity once triggered. That's... terrible! :scream: